app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java

```diff
@@ -232,6 +250,7 @@ public void onDrawerSpecialItemClick(MenuItem item) {
                 Bundle special = new Bundle();
                 special.putString(Const.EXTRA_TAGGED_EVENT_CACHEKEY, taggedEventSeries.getTag());
                 special.putParcelable(Const.EXTRA_TAGGED_EVENT, taggedEventSeries);
+                special.putString(Const.DRAWER_ITEM, item.getTitle().toString());
                 navigateTo(TaggedEventSeriesActivity.class, special);
```

Comment:
Here we could have a check as well if we are already in this activity and have the same tagged event series.